packages/web-components/src/components/layout/__stories__/layout.stories.ts

```diff
@@ -0,0 +1,95 @@
+              odio ut. Varius sit amet mattis vulputate. Enim ut tellus elementum sagittis vitae et leo. Posuere ac ut consequat
+              semper viverra nam libero. Habitant morbi tristique senectus et netus et malesuada fames. Enim neque volutpat ac
+              tincidunt vitae semper quis lectus nulla.
+            </p>
+          </div>
+          <div class="bx--layout-1-3">
+            <h3>layout 1-3</h3>
+            <ul>
+              <li>item 1</li>
+              <li>item 2</li>
+              <li>item 3</li>
+            </ul>
+            <p>
+              A pellentesque sit amet porttitor. Sed euismod nisi porta lorem. Pellentesque dignissim enim sit amet venenatis urna
+              cursus eget nunc. Eu lobortis elementum nibh tellus molestie nunc non blandit massa.
+            </p>
+          </div>
+        </div>
+        <div class="bx--row">
+          <div class="bx--layout-1-3">
+            <h3>layout 1-3</h3>
+            <ul>
+              <li>item 1</li>
+              <li>item 2</li>
+              <li>item 3</li>
+            </ul>
+            <p>
+              A pellentesque sit amet porttitor. Sed euismod nisi porta lorem. Pellentesque dignissim enim sit amet venenatis urna
+              cursus eget nunc. Eu lobortis elementum nibh tellus molestie nunc non blandit massa.
+            </p>
+          </div>
+          <div class="bx--layout-2-3">
+            <h3>layout 2-3</h3>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+              aliqua. Ac turpis egestas maecenas pharetra convallis posuere. Ultrices dui sapien eget mi proin sed libero enim.
+              Elementum facilisis leo vel fringilla. Sed tempus urna et pharetra pharetra massa massa ultricies mi. Nascetur
+              ridiculus mus mauris vitae ultricies leo integer. Eget mauris pharetra et ultrices. Urna neque viverra justo nec
+              ultrices dui sapien. Augue eget arcu dictum varius duis. Eget mauris pharetra et ultrices neque ornare aenean.lorum
+              ipsum
+            </p>
+            <p>
+              A pellentesque sit amet porttitor. Sed euismod nisi porta lorem. Pellentesque dignissim enim sit amet venenatis urna
+              cursus eget nunc. Eu lobortis elementum nibh tellus molestie nunc non blandit massa. Risus at ultrices mi tempus
+              imperdiet nulla malesuada pellentesque elit. Est ultricies integer quis auctor elit sed. Quis risus sed vulputate
+              odio ut. Varius sit amet mattis vulputate. Enim ut tellus elementum sagittis vitae et leo. Posuere ac ut consequat
+              semper viverra nam libero. Habitant morbi tristique senectus et netus et malesuada fames. Enim neque volutpat ac
+              tincidunt vitae semper quis lectus nulla.
+            </p>
+          </div>
```

Comment:
Given decorator is for shared content instead of the story content - Can we move the content here to the story?

Reply:
Hey @asudoh ! moved the decorator content to the story and added the list classes as you recommended

Reply:
Thanks @andysherman2121! And my apologies that I didn't realize the entire content represents layout demo... Is it possible for you to move the top-level `<div>` to the story, too?

Reply:
@asudoh sure! updating now